### ISSUE TITLE
Date Picker Dialog Example up/down arrow issues with NVDA

### ISSUE THREAD

**Issue by Splaktar** [w3c/aria-practices#1272]
The Date Picker Dialog: Date Grid in the [Date Picker Dialog Example](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/datepicker-dialog.html) does not have the expected behavior in NVDA `2019.2.1` when using the arrow keys.

## Up/down keys

Expected
- Up Arrow - Moves focus to the same day of the previous week.
- Down Arrow - Moves focus to the same day of the next week.

Observed
- Up Arrow - Moves focus to the previous day.
- Down Arrow - Moves focus to the next day.

**Comment by JAWS-test** [comment]
The reason for this is that NVDA does not switch to form mode. This prevents the entire operation: Arrow keys up down right left, Home, End, PageUp and PageDown. The problem also occurs with the grid examples (https://w3c.github.io/aria-practices/#grid), whenever the focus is not on the cell, but on the interactive element in the cell (button, links etc.). 
In the calendar popup, the cell is never focused, but always the button in the cell.

The problem does not occur with JAWS.

So it seems to me to be a problem with NVDA because the grids are implemented correctly according to the ARIA specification. Therefore, a ticket should be created at https://github.com/nvaccess/nvda

**Comment by mcking65** [comment]
@JAWS-test is correct. If the user focuses an element in a grid, NVDA should switch into focus mode and stay in focus mode while navigating. Unless, of course, user presses NVDA+space to intentionally go back to browse mode.

This is the kind of ARIA support issue we intend to systematically raise as part of the w3c/aria-at project. In 1 year, if all goes well, we will have screen reader support tables on most or all the APG examples.

Since this is an NVDA bug, I am closing with the "not for practices" label.
